### DISCUSSION TITLE
Manuell geschaltete Geräte

### DISCUSSION THREAD

**Discussion by georg138** [camueller/SmartApplianceEnabler#211] [Q&A]
Gibt es die Möglichkeit, manuell geschaltete Geräte (nicht smarte) sinnvoll einzubinden?
Folgendes Szenario: Ich habe eine Waschmaschine, deren Verbrauch und Zustand (an/aus) über meine Smart-Home Software erfasst wird. Diese Daten über http im SAE abzufragen ist kein Problem, ebenso können so auch Schaltbefehle gesendet werden. Jetzt kann die Waschmaschine aber nicht automatisch geschaltet werden, sondern nur von Hand gestartet werden (nach dem Trennen vom Strom vergisst sie alles). Meine Idee wäre, dass wenn der SHM den Befehl zum Einschalten and den SAE sendet, der SAE den Befehl zum Einschalten über http sendet. Ich bekomme dann von meinem Smart-Home eine Mitteilung, dass ich einschalten soll. Erst wenn das passiert ist, sollte die Zeit im SAE runtergezählt werden. Aktuell wird zwar der Schaltzustand ans SHM übermittelt, aber die Zeit läuft sofort runter. Wenn ich also eine halbe Stunde brauche, weiß das SHM nicht mehr, wie lange die Maschine noch braucht.
Ähnlich könnte man das z.B. für eine Spülmaschine nutzen. Wenn ich die Nachricht bekomme, dass ich die Maschine anschalten soll, kann ich schauen, ob sie halbwegs voll ist. Wenn nicht, lasse ich sie aus, wenn ja starte ich sie. Also eher als Empfehlung, falls gerade Strom übrig ist.
Lässt sich soetwas umsetzen oder würde das SHM damit durcheinander kommen, wenn die Geräte erst später oder gar nicht eingeschaltet werden?

**Comment by camueller** [comment]
Schaltbefehle machen nur Sinn, wenn der SAE das Gerät tatsächlich selbst schalten kann. Ich würde einfach gar keine Schalter im SAE konfigurieren, dann werden einfach die Verbräuche erfasst und Du kannst manuell schalten.
